pkg/mcs/resourcemanager/server/manager.go

```diff
@@ -157,6 +157,20 @@ func (m *Manager) getKeyspaceResourceGroupManager(keyspaceID uint32) *keyspaceRe
 	return m.krgms[keyspaceID]
 }
 
+func (m *Manager) accessKeyspaceResourceGroupManager(keyspaceID uint32, groupName string) (*keyspaceResourceGroupManager, error) {
```

Comment:
Do we need to add it to `GetKeyspaceServiceLimiter`?

Reply:
No need, we can return "Not found" error if the keyspace is not inited.